include/functions.h

```diff
@@ -1836,40 +1836,40 @@ void Math_Vec3f_Scale(Vec3f* vec, f32 scale);
-void IChain_Apply_s8(u8* actor, InitChainEntry* init);
-void IChain_Apply_u16(u8* actor, InitChainEntry* init);
-void IChain_Apply_s16(u8* actor, InitChainEntry* init);
-void IChain_Apply_u32(u8* actor, InitChainEntry* init);
-void IChain_Apply_s32(u8* actor, InitChainEntry* init);
-void IChain_Apply_f32(u8* actor, InitChainEntry* init);
-void IChain_Apply_f32div1000(u8* actor, InitChainEntry* init);
-void IChain_Apply_Vec3f(u8* actor, InitChainEntry* init);
-void IChain_Apply_Vec3fdiv1000(u8* actor, InitChainEntry* init);
-void IChain_Apply_Vec3s(u8* actor, InitChainEntry* init);
-f32 Math_SmoothScaleMaxMinF(f32* a0, f32 a1, f32 a2, f32 a3, f32 a4);
-void Math_SmoothScaleMaxF(f32* start, f32 target, f32 scale, f32 maxStep);
-void Math_SmoothDownscaleMaxF(f32* start, f32 scale, f32 maxStep);
-s32 Math_SmoothScaleMaxMinS(s16* start, s16 target, s16 scale, s16 maxStep, s16 minStep);
-void Math_SmoothScaleMaxS(s16* start, s16 target, s16 scale, s16 maxStep);
+s16 Math_Vec3f_Yaw(Vec3f* a, Vec3f* b);
+s16 Math_Vec3f_Pitch(Vec3f* a, Vec3f* b);
+void Actor_ProcessInitChain(Actor* actor, InitChainEntry* ichain);
+// void IChain_Apply_u8(u8* actor, InitChainEntry* init);
+// void IChain_Apply_s8(u8* actor, InitChainEntry* init);
+// void IChain_Apply_u16(u8* actor, InitChainEntry* init);
+// void IChain_Apply_s16(u8* actor, InitChainEntry* init);
+// void IChain_Apply_u32(u8* actor, InitChainEntry* init);
+// void IChain_Apply_s32(u8* actor, InitChainEntry* init);
+// void IChain_Apply_f32(u8* actor, InitChainEntry* init);
+// void IChain_Apply_f32div1000(u8* actor, InitChainEntry* init);
+// void IChain_Apply_Vec3f(u8* actor, InitChainEntry* init);
+// void IChain_Apply_Vec3fdiv1000(u8* actor, InitChainEntry* init);
+// void IChain_Apply_Vec3s(u8* actor, InitChainEntry* init);
+f32 Math_SmoothStepToF(f32* pValue, f32 target, f32 fraction, f32 step, f32 minStep);
+void Math_ApproachF(f32* pValue, f32 target, f32 scale, f32 maxStep);
+void Math_ApproachZeroF(f32* pValue, f32 scale, f32 maxStep);
+s32 Math_SmoothStepToS(s16* pValue, s16 target, s16 scale, s16 step, s16 minStep);
+void Math_ApproachS(s16* pValue, s16 target, s16 scale, s16 maxStep);
 void Color_RGBA8_Copy(Color_RGBA8* dst, Color_RGBA8* src);
-void func_801000A4(u16 param_1);
-void func_801000CC(u16 param_1);
-void func_801000F4(UNK_TYPE4 param_1, u16 param_2);
-void Lib_TranslateAndRotateYVec3f(Vec3f* translation, s16 rotation, Vec3f* src, Vec3f* dst);
+void func_801000A4(u16 sfxId);
+void func_801000CC(u16 sfxId);
+void func_801000F4(s32 a0, u16 a1);
+void Lib_Vec3f_TranslateAndRotateY(Vec3f* translation, s16 a, Vec3f* src, Vec3f* dst);
 void Lib_LerpRGB(Color_RGB8* a, Color_RGB8* b, f32 t, Color_RGB8* dst);
-f32 Lib_PushAwayVec3f(Vec3f* start, Vec3f* pusher, f32 distanceToApproach);
+f32 Math_Vec3f_StepTo(Vec3f* start, Vec3f* target, f32 speed);
 void Lib_Nop801004FC(void);
 void* Lib_PtrSegToVirt(void* ptr);
 void* Lib_PtrSegToVirtNull(void* ptr);
```

Comment:
Additionally, this is solely my opinion but I believe a better naming scheme for these functions (as well as the `PtrSegToK0` functions just below these) would be names that directly mirror the macros. i.e. `Lib_SegmentedToVirtual` and `Lib_PhysicalToVirtual`